utils/scripts/create_parameter_json_from_nfcore_schema.py

```diff
@@ -0,0 +1,67 @@
+#!/bin/env python3
+
+"""
+Generates a parameter json for AWS HealthOmics workflows
+from NF-CORE nextflow schema JSON.
+"""
+
+from argparse import ArgumentParser
+import json
+from pprint import pprint
+import logging
+
+# Create a logger
+logging.basicConfig(format="%(name)s - %(asctime)s %(levelname)s: %(message)s")
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+
+parser = ArgumentParser()
+parser.add_argument('--nfcore_schema_file', type=str, required=True,
+                    help="nextflow schema JSON used in NF-CORE nextflow workflows")
+parser.add_argument('--output_file', type=str, required=True,
```

Comment:
Suggest having this be an optional parameter with a default of `parameter-template.json`. This way, users can choose if they need to override the output.

Reply:
yup thats a good idea